include/SVF-LLVM/LLVMUtil.h

```diff
@@ -179,10 +176,12 @@ inline bool ArgInDeadFunction (const Value*  val)
 //@}
 
 /// Return true if this is an argument of a program entry function (e.g. main)
-inline bool ArgInProgEntryFunction (const Value*  val)
+inline bool ArgInProgEntryFunction(const Value* val)
 {
-    return SVFUtil::isa<Argument>(val)
-           && SVFUtil::isProgEntryFunction(SVFUtil::cast<Argument>(val)->getParent());
+    return SVFUtil::isa<Argument>(val) &&
+           SVFUtil::isProgEntryFunction(
+               LLVMModuleSet::getLLVMModuleSet()->getSVFFunction(
```

Comment:
Do not use llvm module here, define a isProgEntryFunction in llvm util

Reply:
Added another `llvm::Function` version in `LLVMUtil.h`.